pheps/phep-9999.md

```diff
@@ -0,0 +1,112 @@
+
+# Rationale
+<a name="rationale"></a>
+Decisions for tiering levels, requirements for each tier, and benefits accrued at each tier are based on conversations with the community (bi-annual meetings, telecons, etc.), and are listed here as a starting point for more discussion, likely to be refined in the future. Initially, ideas were presented to the community in a pyramid format. To make the differences between tiers more visible and understandable, it has been transformed into a spreadsheet format. 
+
+# PyHC Package Tiering Specifications
+<a name="specification"></a>
+There are four tiers proposed in this PHEP: Honorable Mention, Bronze, Silver, and Gold. See the table below for requirements associated with each tier:
+
+| Tier                  | Self Evaluation Status | PyHC Standard Grades              | pyOpenSci Review Status | PyHC Env Installation Conflicts | DOI          | Interoperability Status                         | pip Installable? | conda Installable? |
+| :--:                  | :--------------------: | :------------------:              | :---------------------: | :-----------------------------: | :-:          | :---------------------:                         | :--------------: | :----------------: |
+| **Gold**              | Completed              | Mostly green, some yellow allowed | Completed               | No conflicts allowed            | Required     | Interoperable with all other PyHC core packages | Yes              | Yes                |
+| **Bronze**            | Completed              | Several yellow, no red            | In Progress             | A couple conflicts exist        | Required     | Interoperable with most PyHC core packages      | Yes              | No                 |
+| **Silver**            | Completed              | Red grades allowed                | Not Completed           | Major conflicts exist           | Required     | Interoperable with 1-2 PyHC core packages       | Yes              | No                 |
+| **Honorable Mention** | Not Done               | N/A                               | N/A                     | Major conflicts exist           | Not required | Does not interoperate with core packages        | No               | No                 |
+
+Descriptions for each heading are as follows:
```

Comment:
Recommend including links to pages that explain how to do each of these things, to make it easier for new people to fulfill each requirement.

Reply:
Good idea, I'll add that/include in the "How to Teach This" section.